query-engine/driver-adapters/executor/script/bench.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
The script that runs the benchmarks. Feeds the schema into the `bench.ts` program via STDIN